haproxy_exporter.go

```diff
@@ -141,6 +117,59 @@ var (
 		43: newServerMetric("http_responses_total", "Total of HTTP responses.", prometheus.Labels{"code": "5xx"}),
 		44: newServerMetric("http_responses_total", "Total of HTTP responses.", prometheus.Labels{"code": "other"}),
 	}
+
+	frontendMetrics = metrics{
+		4:  newFrontendMetric("current_sessions", "Current number of active sessions.", nil),
+		5:  newFrontendMetric("max_sessions", "Maximum observed number of active sessions.", nil),
+		6:  newFrontendMetric("limit_sessions", "Configured session limit.", nil),
+		7:  newFrontendMetric("sessions_total", "Total number of sessions.", nil),
+		8:  newFrontendMetric("bytes_in_total", "Current total of incoming bytes.", nil),
+		9:  newFrontendMetric("bytes_out_total", "Current total of outgoing bytes.", nil),
+		10: newFrontendMetric("requests_denied_total", "Total of requests denied for security.", nil),
+		12: newFrontendMetric("request_errors_total", "Total of request errors.", nil),
+		33: newFrontendMetric("current_session_rate", "Current number of sessions per second over last elapsed second.", nil),
```

Comment:
What do you think about dropping this pre-computed rate metrics?

Reply:
I see there's a counter `haproxy_server_sessions_total` too, but not sure if that is present in all HAProxy versions?

In this PR I would rather not change the metrics themselves, that would be unnecessarily conflating unrelated changes and possibly breaking people.